nsv13/code/modules/munitions/ship_weapons/ballistic_weapons/deck_guns.dm

```diff
@@ -91,6 +91,9 @@
 		parts[++parts.len] = part
 	data["parts"] = parts
 	data["can_pack"] = core.payload_gate && core.payload_gate.loaded
+	data["strength"] = (core.payload_gate.shell) ? "Packed strength: [core.payload_gate.shell.speed] dT." : null
```

Comment:
So this is extra data you are feeding into the TGUI, yet I don't see any changes to the TGUI.

Reply:
In deckGun.js, you'll need to do something like:
<Fragment>
<Button>$.{data.strength}</Button>
</Fragment>

Reply:
Unused additions removed.